src/features/points/CoLinksGiveButton.tsx

```diff
@@ -41,6 +43,15 @@ export const CoLinksGiveButton = ({
     give => give.giver_profile_public?.id === profileId
   );
 
+  const dismissibleAs = `banner:colinks_give_intro`;
+
+  const [showBanner, setShowBanner] = useState(false);
+
+  const showModalOneTime = () => {
+    setShowBanner(!window.localStorage.getItem(dismissibleAs));
```

Comment:
any reason this isn't just `setShowbanner(false)` ?

Reply:
this pattern seems a bit weird - setting the banner as hidden when you show it. I'd think we should hide it when the user clicks "dismiss" otherwise it dissapears on a reload.